### DISCUSSION TITLE
Someone not active (and with fake project) took our company name  ...

### DISCUSSION THREAD

**Discussion by armandoxxx** [community/community#56748] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Select Feature Area

- [X] Organization

### Body

Hello 
We're a company and we would like to open organization and move our projects to github.
Unfortunately our organization name is taken and we filed a report. 
1st: we did not get any reply regarding the matter. 
2nd: the  [user](https://github.com/dropchop) who took our name was inactive for number of years but now, when we filed a report 3 meeks ago, he imported some bs project to make his account appear like it's an active account. 

Our company name is Dropchop, we are from Slovenia - Europe. We've send all the company paperwork. 
At least give us some reply or feedback of what is going on and if we can get our name back. 
Any help appreciated 
Kind regards 

Armando

**Comment by Gustavonn07** [comment]
Contact GitHub support: If you haven't received a response regarding your report, it's important to reach out to GitHub support directly. Explain the situation in detail, mention that you have submitted all the necessary documentation, and request an update on the status of your report. GitHub's technical support team will be able to provide specialized assistance and guide you accordingly.
link: https://support.github.com/request

Review GitHub's documentation and terms of service: Check GitHub's terms of service and related documentation regarding organization name rules and policies. Make sure you are following all the guidelines and requirements for recovering your organization's name.
link: https://docs.github.com/en

These steps should help you proceed in resolving the issue. Remember that seeking legal advice may also be beneficial if the situation doesn't get resolved through regular channels. Good luck, and I hope you are able to recover your organization's name!

**Reply by airtower-luna** [reply]
Yep, contacting support is probably best. Make sure to read their trademark policy first: https://docs.github.com/en/site-policy/content-removal-policies/github-trademark-policy
